### PR TITLE
[docs] [orc8r] update swagger_ui endpoints on docs

### PR DESCRIPTION
**Pull request by AndyLKhuu**
Signed-off-by: Andy Lee Khuu <andykhuu@stanford.edu>

## Summary
This PR updates our docusarus to reflect the change in the Swagger UI route.
Screen Shot 2021-03-01 at 10 22 00 PM
Screen Shot 2021-03-01 at 10 22 23 PM
Screen Shot 2021-03-01 at 10 22 49 PM

## Test Plan
- [x] Ran `./create_docusaurus_website.sh` and confirmed doc changes

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/feg/deploy_install.md [hunk 1]
```diff
@@ -132,14 +132,14 @@ This will output a hardware ID and a challenge key. This information must be
 registered with the Orchestrator. At this time, NMS support for FeG
 registration is still in-progress.
 
-To register the FeG, go to the Orchestrator's APIdocs in your browser. 
-**Note: It is highly encouraged to use V1 of the apidocs**
-(i.e. https://controller.url.sample:9443/apidocs/v1/).
+To register the FeG, go to the Orchestrator's Swagger UI in your browser.
+**Note: It is highly encouraged to use V1 of the Swagger UI**
```
Comment (hcgatewood) [inline docs/readmes/feg/deploy_install.md]: Remove this note

File: docs/readmes/faq/magma_faq.md [hunk 2]
```diff
@@ -91,15 +91,15 @@ This section lists some of the commonly asked questions related to Magma operati
   - Login to Gateway, then run `sudo checkin_cli.py`.
 
 ### Where can I find API endpoint?
-  - If you have followed the [install guide](https://magma.github.io/magma/docs/orc8r/deploy_install), it'll be at `https://api.youdomain.com/apidocs/v1/` (the trailing slash is important).
+  - If you have followed the [install guide](https://magma.github.io/magma/docs/orc8r/deploy_install), it'll be at `https://api.youdomain.com/swagger/v1/ui`.
```
Comment (hcgatewood) [inline docs/readmes/faq/magma_faq.md]: Here and elsewhere -- let's keep the trailing slash for the monolithic endpoints

**Review by hcgatewood — APPROVED** [review]
Handle comments then msg to merge

**Review by themarwhal — APPROVED** [review]
Looks good! It might be useful to include the `?docExpansion=none#/` postfix I find that tremendously useful, but could be in another PR.

**Comment by AndyLKhuu** [comment]
> Looks good! It might be useful to include the `?docExpansion=none#/` postfix I find that tremendously useful, but could be in another PR.

Noted! I'll make sure to add that in a subsequent PR. Planning to make an overarching one detailing Swagger UI in depth.

**Pull request merged**